ui/src/components/SchemaTable/index.jsx

```diff
@@ -119,14 +107,18 @@ export default class SchemaTable extends Component {
     }
   }
 
-  readReference(schemaPath) {
-    const { serviceName } = this.props;
-    const id = schemaPath.startsWith('/')
-      ? schemaPath
-      : join('/', 'schemas', serviceName, schemaPath);
-    const schemaId = id.endsWith('#') ? id : `${id}#`;
+  readReference(schemaId) {
+    const fullSchemaId =
+      schemaId.indexOf('#') === -1 ? `${schemaId}#` : schemaId;
```

Comment:
you can also do `schemaId.includes('#') ? ` <=== if you find that's more readable

Reply:
Oh, hm, I meant to preserve existing behavior there (only add a # at the *end*).  I'm not sure why I changed that.. So I'll switch back to the original `endsWith`.